static/js/popover_menus.js

```diff
@@ -52,7 +52,17 @@ export function any_active() {
 }
 
 function on_show_prep(instance) {
-    $(instance.popper).one("click", instance.hide);
+    $(instance.popper).on("click", (e) => {
+        // We don't hide the popover here since we want the element being clicked to
+        // handle it completely. Also, this avoids popover being closed on clicks
+        // in popover which are not inside interactive elements.
+        e.stopPropagation();
+    });
```

Comment:
Is this block required? Certainly the comment seems misleading, since we no longer do a hide on clicking on the popover.

Reply:
Sorry my comment wasn't clear, updated!